manifests/5.1.beta1/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -96,10 +96,6 @@ spec:
                       properties:
                         groupBy:
                           description: GroupBy defines the strategy for grouping logstreams
-                          enum:
```

Comment:
looks like the validations are still missing from the CRD.  `make check` maybe?

Reply:
Sorry. Fixed.